jvm/src/test/scala/io/glutenproject/execution/WholeStageTransformerSuite.scala

```diff
@@ -135,14 +135,20 @@ abstract class WholeStageTransformerSuite extends GlutenQueryTest with SharedSpa
     result
   }
 
+  def checkLengthAndPlan(df: DataFrame, len: Int = 100) {
+    assert(df.collect().length == len)
+    assert(df.queryExecution.executedPlan
+      .find(_.isInstanceOf[TransformSupport]).isDefined)
+  }
```

Comment:
@lviiii 
As long as any operator is TransformSupport，this check will pass.
So for some cases，it‘s  not applicable.

cc @rui-mo

Reply:
We can change this function to 

```
```

Reply:
Looks like we can change this function to sth. like below to support any type of verification.

```
  def checkLengthAndPlan[TransformerType](df: DataFrame, len: Int = 100) {
    assert(df.collect().length == len)
    assert(df.queryExecution.executedPlan
      .find(_.isInstanceOf[TransformerType]).isDefined)
  }
```

When calling this function, we need to put the type of plan we want to verify. eg. checkLengthAndPlan[ProjectTransformer](df, 1).

Reply:
@zhixingheyi-tian @lviiii How do you think so?

Reply:
I think length check can  be omitted. Because runQueryAndCompare has include this check, comparing Vanilla.
Plan check  can reuse  here  check function.

![image](https://user-images.githubusercontent.com/41657774/200213407-1fd90519-eafb-4269-9360-71c1164e2795.png)

Reply:
> @zhixingheyi-tian @lviiii How do you think so?

I agree.

Reply:
> I think length check can be omitted. Because runQueryAndCompare has include this check, comparing Vanilla. Plan check can reuse here check function.
> 
> ![image](https://user-images.githubusercontent.com/41657774/200213407-1fd90519-eafb-4269-9360-71c1164e2795.png)

Yes, you can omit the length check and reuse method "runQueryAndCompare".